Resources/translations/messages.en.xlf

```diff
@@ -245,6 +245,10 @@
         <source>File not saved on database</source>
         <target>File not saved on database</target>
       </trans-unit>
+      <trans-unit id="k_4Bpb1" resname="Maximum title allowed for playlist are 150 characters.">
+        <source>Maximum title allowed for playlist are 150 characters.</source>
```

Comment:
The maximum size allowed for the playlist title is 150 characters.

Reply:
Done see: a63cace5f13008f226c11fb3f719a714292b63ed